_dashboards/management/multi-data-sources.md

```diff
@@ -195,11 +195,41 @@ When the data source feature is enabled, you can navigate to **Notifications** u
+When the data source feature is enabled, you can navigate to **Search Relevance** under the **OpenSearch Plugins** menu. On the navigation bar on the upper right, you can view all available data sources. To compare search results between indexes from different data sources, first select a data source and an index for **Query 1**, and then select a data source and an index for **Query 2**. Select **Search** to run the queries. The following GIF illustrates these steps.
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboards/searchrelevance_mds1.gif" alt="Multiple data sources in Search Relevance plugin"/>
 
-## Next steps 
+
+### Security analytics
+
+When the data source feature is enabled, you can navigate to **Security analytics** under the **OpenSearch Plugins** menu. Using detection rules as an example, you can view all connected data sources and select a specific one from the navigation bar on the upper right. By default, the detection rules from the designated default data source are displayed. However, you can select any connected data source to view its corresponding detection rules. The following GIF illustrates these steps.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/mds_sa_detection_rules_view.gif" alt="Multiple data sources in Security analytics list page"/>
+
+To create a new detection rule, select the **Create detection rule** button, which opens a form. Fill in the required information and select the **Create detection rule** button. The detection rule is created within the selected data source. The following GIF illustrates these steps.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/mds_sa_detection_rules_create.gif" alt="Multiple data sources in Security analytics create page"/>
+
+
+### Alerting
+
+When the data source feature is enabled, you can navigate to **Alerting** under the **OpenSearch Plugins** menu and view all connected data sources. In the data source view, the default data source and its corresponding detection rules are displayed. However, you can select any connected data source to view its corresponding detection rules from the **Data sources** dropdown. The following GIF illustrates these steps using detection rules.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/mds_monitor_view.gif" alt="Multiple data sources in Alerting list page"/>
+
+To create a new monitor, select the **Create monitor** button, which opens a form. Fill in the required information and select the **Create** button. The monitor is created within the selected data source.
+
+### Feature anywhere
+
+When the data source feature is enabled, you can navigate to **Dashboards** under the **OpenSearch Dashboards** menu. Select a dashboard from the list, select the **Option** button on the dashboard. Then, select  **Alerting** > **Associated monitors**. You should see the associated monitors for the selected dashboard. The following GIF illustrates these steps.
```

Comment:
```suggestion
When the data source feature is enabled, you can access associated monitors for a specific dashboard through OpenSearch Dashboards. Navigate to the **Dashboards** application under the OpenSearch Dashboards menu. From the list of available dashboards, select the one you want to view associated monitors for. Once the dashboard is open, select the **Option** button located on the dashboard itself. This will reveal a set of options, including **Alerting**. Under the **Alerting** option, choose **Associated monitors**. Upon selecting this, you should see the associated monitors displayed for the currently selected dashboard. The following GIF illustrates these steps.
```